v4l2.go

```diff
@@ -311,6 +311,24 @@ func getFrameSize(fd uintptr, index uint32, code uint32) (frameSize FrameSize, e
 	return
 }
 
+func getName(fd uintptr) (string, error) {
+	caps := &v4l2_capability{}
```

Comment:
```suggestion
	var caps v4l2_capability
```

and then do pointer to below

Reply:
I opted to keep things consistent with the other code in this file. I don't think this changes the behavior at all but let me know if that isn't true.

Reply:
Let's go with the suggested change since it's better/common style.